batch/batch/cloud/gcp/driver/pricing.py

```diff
@@ -268,12 +447,43 @@ def process_disk_sku(sku: dict, regions: List[str]) -> List[GCPDiskPrice]:
     return disk_prices
 
 
+# Not called but this logic is how I got the gpu SKUs
+async def get_gpu_skus(billing_client: aiogoogle.GoogleBillingClient, currency_code: str) -> List[set]:
+    params = {'currencyCode': currency_code}
+    g2_ram_total = set()
+    g2_cpu_total = set()
+    gpu_total = set()
+    async for sku in billing_client.list_skus('/6F81-5844-456A/skus', params=params):
+        category = sku['category']
+        if (
+            sku['description'].startswith("G2 Instance")
+            or sku['description'].startswith("Spot Preemptible G2 Instance")
+        ) and "us-" in sku['serviceRegions'][0]:
```

Comment:
What is the reason for limiting to only the `us`? Applies below as well.

Reply:
I think maybe we will only roll out GPUs in the US for now and not in the Australia deploy